src/Footer/Footer.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Footer from './Footer';
+
+const setup = () => {
+  const component = shallow(<Footer />);
+  return component;
+};
+
+describe('Footer component', () => {
+  let component;
+  beforeEach(() => {
+    component = setup();
+  });
+
+  it('should render Component', () => {
```

Comment:
`should be rendered without errors` sounds better

Reply:
Corrected